lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/view/contententry/detailattempttab/ContentEntryDeatilAttemptsSessionListScreen.kt

```diff
@@ -0,0 +1,169 @@
+package com.ustadmobile.libuicompose.view.contententry.detailattempttab
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Icon
```

Comment:
All material imports must use material3 e.g. material3.Text material3.Icon , etc. Otherwise theme/appearance will be inconsistent.

Reply:
Note there is no material3.icons (vectors) - this applies to Components - e.g. Text, ListItem, Icon, etc.

Reply:
Used import of material3 for Text,Icon,ListIem,etc